src/main/java/org/folio/converters/AddressConverter.java

```diff
@@ -0,0 +1,29 @@
+package org.folio.converters;
+
+import org.folio.rest.acq.model.Address;
+import org.folio.rest.jaxrs.model.VendorAddress;
+import org.springframework.core.convert.converter.Converter;
+import org.springframework.stereotype.Service;
+
+@Service
```

Comment:
Configuring beans via Java config is preferable to config via annotations.
Could you please move this to : ServicesConfiguration?

Reply:
Thank you for your input. I was confused by the use of `ServicesConfiguration` in `VoucherService` as opposed to the use of `@Autowired` and `SpringContextUtil.autowireDependencies` in `BatchVoucherGenerateService`. I modified the code to be more consistent with the existing classes, and avoid a potential issue with the missing `SpringContextUtil.autowireDependencies`.